bot.py

```
@@ -104,7 +107,9 @@ async def info(message: Message, bot: Bot):
         "Отзыв🆕: Теперь вы можете оставить отзыв про бота!\n"
         "*Память*🧠: *Публичный предпросмотр*\n"
         "*Расшифровка голосовых сообщений в текст*:\n"
-        "Просто отправьте или перешлите голосовое сообщение и я его расшифрую\n\n"
+        "Просто отправьте или перешлите голосовое сообщение и я его расшифрую\n"
```

Ruff (0.12.2)

String contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF001)